src/sentry/api/endpoints/project_template_detail.py

```diff
@@ -61,3 +63,39 @@ def delete(self, request: Request, organization: Organization, template_id: str)
 
         # think about how to handle there being no remaining templates for an org?
         return Response(status=204)
+
+    def put(self, request: Request, organization: Organization, template_id: str) -> Response:
+        """
+        Update the project template name or options.
+
+        Return the updated project template.
+        """
+        serializer = ProjectTemplateWriteSerializer(data=request.data, partial=True)
+        if not serializer.is_valid():
+            return Response(status=400, data=serializer.errors)
+
+        data = serializer.validated_data
+
+        project_template = get_object_or_404(
+            ProjectTemplate, id=template_id, organization=organization
+        )
+
+        name = data.get("name", None)
+        if name is not None:
+            project_template.name = name
+            project_template.save()
+
+        options = data.get("options", {})
```

Comment:
I don't think you need the default empty dict here if you're checking it on the next line

Reply:
d'oh, looks like there was a merge conflict.

the way i had this before the merge conflict was to not have the `if` statement on the next line, and just use the `for` loop `.items` condition to limit the creation of options. thoughts on that approach vs removing the default value here?

Reply:
I like this better 👍